services/ui_backend_service/api/utils.py

```diff
@@ -20,6 +45,14 @@ def get_json_from_env(variable_name: str):
         return None
 
 
+def get_json_from_file(filepath: str):
+    try:
+        with open(filepath) as f:
+            return json.load(f)
+    except Exception:
+        return None
```

Comment:
In #283 I added more logging into when there is an exception. I think that would be helpful here too

Reply:
Added some comments to the other PR regarding logging. Added logging to the config file loader here as suggested, with the caveat that we're only interested in cases where a file is loaded (found), but something goes wrong.

a config file not existing, or a config env_var not being set is not a reportable incident in my opinion, as these are optional configs